openai_functools/functions_orchestrator.py

```diff
@@ -9,20 +9,24 @@
 
 
 class FunctionsOrchestrator:
+    SUPPORTED_MODEL_VERSIONS = ["gpt-3.5-turbo", "text-davinci-002", "text-curie-003"]
```

Comment:
Should we add the version number to this list, because we might not support future version? (e.g. "gpt-3.5-turbo-0613")

We can then also optionally verify when someone wants to use a feature whether it's actually available for the current selected model version.